SU-HSDO/suhumsci
# SHS-5782: Vertical button card fix text gets truncated / SHS-5793: Implementation: Vertical Button Card button is currently styled as a rectangle

# READY FOR REVIEW

## Summary
Fix the button text getting truncated in Vertical Button Card in Colorful
Fix currently button style in Vertical Button Card in Colorful
Fix issue when Vertical Button Card with no title or no description

## Need Review By (Date)
09/04

## Urgency
medium

## Steps to Test
1. Go to [Colorful](https://hs-colorful-5twp4oteg8kiutpmqkq0r9agn2688xbx.tugboatqa.com/components/collections/collections-postcards/vertical-button-cards-0/vertical-button-cards-0)
2. Check button style looks according to what's in the [design](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=4497-3745&t=gfAySXTRsOeLNSB3-4)
3. Check button text now forces hyphenation and respect the margins
4. Check that Vertical Button Card looks good in [these different scenarios](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=4738-155&t=gfAySXTRsOeLNSB3-4)
5. Now go to [Traditional](https://hs-traditional-5twp4oteg8kiutpmqkq0r9agn2688xbx.tugboatqa.com/components/collections-0/collections-postcards/vertical-button-cards/vertical-button-cards-well)
6. And check that Vertical Button Card looks good in [these different scenarios](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=4790-440&t=gfAySXTRsOeLNSB3-4)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208192527102423
